houghtool/__init__.py

```diff
@@ -609,25 +615,30 @@ def intersection(line1, line2, tolerance=1e-6, subpixel=False):
 
 
 def draw_lines(img, lines, color=None, thickness=1):
+    num_lines_drawn = 0
     h, w = img.shape[:2]
     color = color or (0, 255, 0) if len(img.shape) == 3 else 255
     _linetype = linetype(lines[0])
     if _linetype == RHOTHETA:
         lines = [convert(line, bbox=[0, 0, w, h]) for line in lines]
     for line in lines:
         for x1, y1, x2, y2 in line:
-            if ERRORVAL in [x1, y1, x2, y2]:
-                print("Line {} has np.nan vals; not drawing.".format(line))
+            # if ERRORVAL in [x1, y1, x2, y2]:
+            if True in np.isnan([x1, y1, x2, y2]):
+                # print("Line {} has np.nan vals; not drawing.".format(line))
                 continue
             cv2.line(img, (x1, y1), (x2, y2), color, thickness)
+            num_lines_drawn += 1
+    print('drawn %u lines' % num_lines_drawn)
     return img
 
 
 def mark_endpoints(img, lines, color=None, markerType=cv2.MARKER_CROSS, markerSize=5):
     color = color or (0, 255, 255) if len(img.shape) == 3 else 255
     for line in lines:
         for x1, y1, x2, y2 in line:
-            if ERRORVAL in [x1, y1, x2, y2]:
+            # if ERRORVAL in [x1, y1, x2, y2]:
+            if True in np.isnan([x1, y1, x2, y2]):
```

Comment:
As an FYI, instead of checking `True in <iterable>` you can use the built-in `any(<iterable>)`. Similarly numpy has an `any()` method on arrays, so it's probably most clean to do either `if any(np.isnan(...))` or `if np.isnan(...).any()`

Reply:
Thanks for sharing, I agree with your suggestion.